src/main/scala/nl.knaw.dans.easy.deposit/servlets/package.scala

```diff
@@ -132,12 +166,22 @@ package object servlets extends DebugEnhancedLogging {
       matchesEitherOf(archiveExtRegexp, archiveContentTypeRegexp)
     }
 
-    def getArchiveInputStream: Try[resource.ManagedResource[ArchiveInputStream]] = Try {
+    private def getArchiveInputStream: Try[resource.ManagedResource[ArchiveInputStream]] = Try {
       val charSet = fileItem.charset.getOrElse("UTF8")
       if (matchesEitherOf(s".+[.]$tarExtRegexp", tarContentTypeRegexp))
-        managed(new TarArchiveInputStream(fileItem.getInputStream, charSet))
+        managed(new TarArchiveInputStream(fileItem.getInputStream, 10240, 512, charSet, true))
```

Comment:
Could you please add a comment about why you choose these specific numbers?
Especially because these appear to be the defaults in other constructors of `TarArchiveInputStream`. If it's because the appropriate constructor doesn't exist, please use the constants instead.

```suggestion
        managed(new TarArchiveInputStream(fileItem.getInputStream, TarConstants.DEFAULT_BLKSIZE, TarConstants.DEFAULT_RCDSIZE, charSet, true))
```